test/NinjaTools.FluentMockServer.API.Tests/Models/RequestBodyMatcherTests.cs

```diff
@@ -0,0 +1,103 @@
+using System.IO;
+using System.Text;
+using FluentAssertions;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using NinjaTools.FluentMockServer.API.Models;
+using NinjaTools.FluentMockServer.Tests.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NinjaTools.FluentMockServer.API.Tests.Models
+{
+    public class RequestBodyMatcherTests : XUnitTestBase<RequestBodyMatcherTests>
+    {
+        /// <inheritdoc />
+        public RequestBodyMatcherTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private HttpContext CreateContext([CanBeNull] string textContent = null, byte[] byteContent = null)
+        {
+            if (textContent != null)
+            {
+                byteContent = Encoding.UTF8.GetBytes(textContent);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Introduce a new variable instead of reusing the parameter 'byteContent'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4765163)